test/utility.cpp

```diff
@@ -91,6 +91,129 @@ TEST_CASE("exchange", "[utility.exchange]") {
   }
 }
 
+// [forward], forward/move
+
+// template<class T>
+//   constexpr T&& forward(remove_reference_t<T>& t) noexcept;
+// template<class T>
+//   constexpr T&& forward(remove_reference_t<T>&& t) noexcept;
+TEST_CASE("forward", "[forward]") {
+  struct S {};
+
+  struct {
+    constexpr S operator()() const noexcept { return S{}; }
+  } make_s;
+
+  struct {
+    constexpr S const operator()() const noexcept { return S{}; }
+  } make_cs;
```

Comment:
I'd suggest replacing these with `declval`, but we might end up needing to produce const rvalue instances...

Reply:
Do you mean replacing `make_s` with `declval<S>()` and `make_cs` with `declval<const S>()`?

Reply:
Would replacing all the maker objects with a global `make<T>` function help?